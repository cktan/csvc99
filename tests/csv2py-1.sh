../csv2py -d '|' -n '(null)' in/csv2py-1.csv 
