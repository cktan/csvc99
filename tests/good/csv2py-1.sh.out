[
	['1','2',None,'3'],
	['abcd','efg','hij','klm']
]

